New parsers (#119)
aramis-lab/AD-DL: docs/Train/ROI.md

Choices are `best_loss` and `best_balanced_accuracy`. Default: `best_balanced_accuracy`.
- `--selection_threshold` (float) threshold on the balanced accuracies to compute the 
[image-level performance](./Details.md#soft-voting). 

> Regions are selected if their balanced accuracy is greather than the threshold. Default corresponds to no selection.

Reviewer: ```suggestion
Regions are selected if their balanced accuracy is greater than the threshold. Default corresponds to no selection.
```

    Region-level performance corresponds to an evaluation in which both ROI are considered to be independent.
    However it is not the case, and what is more interesting is the evaluation at the image-level, 
    for which the predictions of the two regions were [assembled](./Details.md#soft-voting).

## `train roi multicnn` - Train one classification CNN per region

> Contrary to the preceding network in which all regions of interest were used as input of a unique CNN, with this option

Reviewer: ```suggestion
Contrary to the preceding network in which all regions of interest were used as input of a unique CNN, this option
```

> a CNN is trained per region. Then the predictions of the CNNs are [assembled](./Details.md#soft-voting) to determine

Reviewer: This suggestion follows the previous one. 

```suggestion
allows to train a CNN per region. Then the predictions of the CNNs are [assembled](./Details.md#soft-voting) to determine
```

the label at the image level.

The output of each CNN is a vector of size equals to the number of classes in this dataset.
This vector can be preprocessed by the [softmax function](https://pytorch.org/docs/master/generated/torch.nn.Softmax.html) 
to produce a probability for each class. During training, the CNN is optimized according to the cross-entropy loss, 

> which becomes null for a subset of images if the CNN outputs 100% probability for the true class of each image of the subset.

Reviewer: I will change this text, to make it more clear:

> "During the training, the CNN is optimized according to the cross-entropy loss. This value becomes zero (or null?) for a subset of images if the probability of the CNN is 1, with respect to the true class (ground truth) of each image in the subset."

Author: Done in https://github.com/aramis-lab/AD-DL/pull/119/commits/ca0ae2054e0ae5f73ab4de3084096140af19f0f9
